Dockerfile

```diff
@@ -0,0 +1,20 @@
+# This is the Dockerfile used by the hadjri/evg-container-self-testing image.
+FROM ubuntu:20.04
+CMD ["/bin/bash"]
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt update
+RUN apt install -y curl
+RUN apt install -y git
+RUN apt install -y make
+RUN apt install -y python
+RUN apt install -y wget
+RUN apt install -y rsync
```

Comment:
Out of my own curiosity, what's `rsync` for?

Reply:
There's a few [tests](https://spruce-staging.corp.mongodb.com/task/evg_ubuntu2004_container_test_operations_patch_be297a810269473f5e295b74e45dd50575580098_646d1de19dbe326f719d0623_23_05_23_20_11_27/tests?execution=0&sortBy=STATUS&sortDir=ASC) in `test-operations` that need it